src/libs/OptionsListUtils.js

```diff
@@ -375,7 +375,10 @@ function getAllReportErrors(report, reportActions) {
  * @returns {String}
  */
 function getLastMessageTextForReport(report) {
-    const lastReportAction = lastReportActions[report.reportID];
+    const lastReportAction = _.find(
+        allSortedReportActions[report.reportID],
+        (reportAction, key) => ReportActionUtils.shouldReportActionBeVisible(reportAction, key) && reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
```

Comment:
Just noting that the use of `shouldReportActionBeVisible` here caused a [regression](https://github.com/Expensify/App/issues/30575) that caused the LHN subtitle to be incorrect (`This is the beginning...`), as `shouldReportActionBeVisible` is really for showing report actions in `ReportActionList`, so `shouldReportActionBeVisibleAsLastAction` is more appropriate here. More details [here](https://github.com/Expensify/App/issues/30575#issuecomment-1788560074).